eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -136,14 +136,51 @@ jobs:
               $outputPath = Join-Path -Path $(Build.ArtifactStagingDirectory) packages/${{ artifact.Name }}/package-info.json
 
               $version = eng/scripts/Get-PkgVersion -ServiceDirectory ${{ parameters.ServiceDirectory }} -PackageName ${{ artifact.Path }}
-              $outputObject = @{ version = $version.ToString(); name = '${{ artifact.Name }}' } | ConvertTo-Json
+              $outputObject = @{ version = $version.ToString(); name = '${{ artifact.Name }}'; packageName = "${{ artifact.Name }}_$version"; } | ConvertTo-Json
 
               Set-Content -Path $outputPath -Value $outputObject
 
               # Set AZURE_SDK_VERSION for use in Doxygen generation
               Write-Host "##vso[task.setvariable variable=AZURE_SDK_VERSION]$version"
             displayName: Create package info JSON file
 
+          # Building from the root of the repo at this point because of the
+          # nature of the CMake build system. The SoureDirectory should probably
+          # change.
+          - task: powershell@2
+            inputs:
+              targetType: filePath
+              filePath: eng/scripts/New-ReleaseArtifact.ps1
+              arguments: >-
+                -SourceDirectory ./
```

Comment:
Given that we are using a relative path here I'd highly suggest we explicitly set the workingDirectory on this step.

Reply:
The working directory I was referring to here is for the Powershell task. I just wanted to be sure that the "." here resolves to something we expect it to instead of whatever random directory this script gets ran in.